src/providers/wfs/qgswfsprovidermetadata.cpp

```diff
@@ -243,14 +245,26 @@ QList<QgsProviderSublayerDetails> QgsWfsProviderMetadata::querySublayers( const
   details.setWkbType( provider.wkbType() );
   res << details;
 
+  // Always issue a GetFeature because the guessed type can't be trusted,
```

Comment:
"Always" doesn't seem appropriate here as it is conditional, unless I misunderstood the comment

Reply:
Yeah, it means:
  // If set always issue a GetFeature because the guessed type can't be trusted,

I'll fix that.

Reply:
fixed